src/Epinova.ElasticSearch.Core/Utilities/Language.cs

```diff
@@ -14,15 +14,41 @@ internal static class Language
         /// </summary>
         private static readonly Dictionary<string, string> AnalyzerMappings = new Dictionary<string, string>
         {
+            { "ar", "arabic" },
+            { "am", "armenian" },
+            { "eu", "basque" },
+            { "bn", "bengali" },
+            { "pt-br", "brazilian" },
+            { "bg", "bulgarian" },
+            { "ca", "catalan" },
+            //{ "", "cjk" },    Don't know the language code for this. If you do feel free to add it.
```

Comment:
it's for Japanese, Korean and Chinese, but the default code to generate the analyzer settings will fail, since CJK requires a bit different setup